common/network-shuffle/src/test/java/org/apache/spark/network/shuffle/ExternalShuffleBlockResolverSuite.java

```diff
@@ -147,22 +147,20 @@ public void jsonSerializationOfExecutorRegistration() throws IOException {
 
   @Test
   public void testNormalizeAndInternPathname() {
-    assertPathsMatch("/foo", "bar", "baz",
-      File.separator + "foo" + File.separator + "bar" + File.separator + "baz");
-    assertPathsMatch("//foo/", "bar/", "//baz",
-      File.separator + "foo" + File.separator + "bar" + File.separator + "baz");
-    assertPathsMatch("foo", "bar", "baz///",
-      "foo" + File.separator + "bar" + File.separator + "baz");
-    assertPathsMatch("/foo/", "/bar//", "/baz",
-      File.separator + "foo" + File.separator + "bar" + File.separator + "baz");
-    assertPathsMatch("/", "", "", File.separator);
-    assertPathsMatch("/", "/", "/", File.separator);
+    String sep = File.separator;
+    String expectedPathname1 = sep + "foo" + sep + "bar" + sep + "baz";
```

Comment:
I meant this value only. The others are used only once.

Reply:
I prefer to write in single style if there is no strict rule that "value used only once must be inlined"

Reply:
Extracting a variable which only used once just increases the indirection. 

Especially here where the role of the value is trivial. If it would be something complex then I might understand to use a describing name but here not.

Reply:
OK, reverted.